### PR TITLE
Add acceptance tests template and minor beautifications

### PR DESCRIPTION
**Pull request by lluiscampos**

### CODE REVIEW

File: .gitlab-ci-check-golang-acceptance.yml [hunk 1]
```diff
@@ -0,0 +1,110 @@
+# .gitlab-ci-check-golang-acceptance.yml
+#
+# This gitlab-ci template build, runs and publises into codecov acceptance
```
Comment (oleorhagen) [inline .gitlab-ci-check-golang-acceptance.yml]: Nitpick: 
s/build/builds
s/publises/publishes

File: .gitlab-ci-check-golang-acceptance.yml [hunk 2]
```diff
@@ -0,0 +1,110 @@
...
+# It assumes a tests/ folder with a Dockerfile for the Python test runner,
+# and a Dockerfile.acceptance-testing for the Go test binary.
+#
+# Add it to the project in hand through Gitlab's include functionality
+#
+# include:
+#   - project: 'Northern.tech/Mender/mendertesting'
+#     file: '.gitlab-ci-check-golang-acceptance.yml'
+#
+# The template uses latest golang version. To override the version
```
Comment (oleorhagen) [inline .gitlab-ci-check-golang-acceptance.yml]: Nitpick: `1.13` is latest not, but not for as long as this comment will live probs

**Comment by lluiscampos** [comment]
Blocks https://github.com/mendersoftware/inventory/pull/148. I tested it in a hackish way [here](https://gitlab.com/Northern.tech/Mender/inventory/pipelines/108890731).

File: .gitlab-ci-check-golang-acceptance.yml [hunk 3]
```diff
@@ -0,0 +1,110 @@
...
+  image: docker
+  services:
+    - docker:dind
+  dependencies:
+    - test:build_acceptance
+  before_script:
+    - export DOCKER_REPOSITORY="mendersoftware/${CI_PROJECT_NAME}"
+    - export DOCKER_TAG=${CI_COMMIT_REF_SLUG:-master}
+    - export SERVICE_IMAGE=$DOCKER_REPOSITORY:$DOCKER_TAG
+    - export COMMIT_TAG="$CI_COMMIT_REF_SLUG"_"$CI_COMMIT_SHA"
```
Comment (oleorhagen) [inline .gitlab-ci-check-golang-acceptance.yml]: Nitpick: 
Simply quote the whole thing mby?

File: .gitlab-ci-check-golang-acceptance.yml (hunk 2 above)
Comment (lluiscampos) [reply .gitlab-ci-check-golang-acceptance.yml]: I don't understand your comment.

Comment (oleorhagen) [reply .gitlab-ci-check-golang-acceptance.yml]: sry. The jist of it is. What happens when they release go `1.14` ?

File: .gitlab-ci-check-golang-acceptance.yml (hunk 3 above)
Comment (lluiscampos) [reply .gitlab-ci-check-golang-acceptance.yml]: Okey. I'll fix in `.gitlab-ci-check-golang-dockerimage.yml` also.

File: .gitlab-ci-check-golang-acceptance.yml (hunk 2 above)
Comment (lluiscampos) [reply .gitlab-ci-check-golang-acceptance.yml]: I see, thanks.

I will rephrase a bit the comment (as using the actual latest with `golang:latest` is very dangerous`).

**Pull request merged**